hack/build.sh

```diff
@@ -51,6 +51,11 @@ if [[ -z "${GIT_TREE_STATE}" ]]; then
     exit 1
 fi
 
+GCFLAGS=""
+if [[ ${DEBUG:-} = "1" ]]; then
```

Comment:
Is there a better way to make a boolean flag in bash?

Reply:
I don't believe there is, but my bash knowledge is admittedly incomplete and sporadic.